internal/engine/image_build_and_deployer.go

```diff
@@ -115,8 +115,10 @@ func (ibd *ImageBuildAndDeployer) BuildAndDeploy(ctx context.Context, st store.R
 
 	startTime := time.Now()
 	defer func() {
-		// todo: image tag?
-		ibd.analytics.Timer("build.image", time.Since(startTime), nil)
+		ibd.analytics.Timer("update", time.Since(startTime), map[string]string{
```

Comment:
are we concerned about breaking the existing graphs?

Reply:
mm... fair question. I assumed we weren't, but if we are, I can certainly leave the old Timer line in there as well

Reply:
as a general rule of thumb, with data work, the teams I've worked with have had a strong bias towards following existing convention. 

Changing the names of things creates a lot of work for any future person that has to analyze the data, or a data engineer that has to clean up the data.

Reply:
Yeah I would advise against changing names of things. We have legacy data (which we could potentially backfill), but also old versions of Tilt in the wild that are sending data using the legacy format. I think a safer approach is adding _new_ data, and leave old things alone.

Reply:
I like the new nicer data because it nests EVERYTHING under `update` so you can query for all `update` events OR check the type of update by filtering on tags (instead of looking for `name == build.image || name == build.container || ...`). But that's a fair point about changing the schema. Here's a version that logs data in both formats.

Reply:
Thanks @maiamcc to responding to my feedback. But sorry I waded into here when I shouldn't have. I'll just say that my desired high-level requirement is to make it as easy and simple as possible to write queries and that they don't break with new changes. I understand that from time to time there are other tradeoffs that make it so this isn't always possible. I'm just happy to be able to run queries here in the first place! So whatever you end up doing, as long as you let me know soon-ish afterward of changes (and that we can keep the [docs](https://github.com/tilt-dev/tilt-warehouse/blob/master/SCHEMA.md#tiltbuildimage) updated), that's good enough for me!

:homer-bushes-gif: